apps/funke/tamagui.config.ts

```diff
@@ -45,9 +46,21 @@ const config = createTamagui({
   fonts: {
     default: fontOpenSans,
     heading: fontRaleway,
+    // Somehow adding body font gives build errors?!
+    // body: fontOpenSans,
   },
   themes: {
-    light: tokens.color,
+    light: {
+      ...tokens.color,
+
+      // Button
+      buttonHeight: 53,
+
+      // Button.Outline
+      buttonOutlineBackgroundColor: hexColors['grey-100'],
+      buttonOutlineBorderColor: '#ebedef',
+      buttonOutlineTextColor: hexColors['grey-900'],
+    },
```

Comment:
> Also ideally we don't set fixed heights on buttons. I mostly used padding to resize it.

Ah okay, took your design too literally then :) 

> What button is this? Can't we do this in the button component itself?

The outline button. We can't really do it as it changes based on the config and it's very different for Paradym and Funke.

Reply:
But if you have suggestions / concrete implementation proposal I'm all ears

Reply:
Ah this is the button with the icon? And i'm guessing the icon makes it higher than the regular button with only text.

For one-time overrides like this i'd pass in the height as a prop where you use it. It's often better than cluttering your config with values (imo).

The colouring is harder. Again you could use a prop override, but you don't want to do that all the time. As you also have the height override, it might be better to just make a new button variant that fits the requirements.

Reply:
> And i'm guessing the icon makes it higher than the regular button with only text.

No not necessarily . I just set the button height for all buttons as it was very specific in hte design and the same height everywhere. Didn't know it was meant to be set with padding. So will update it.


For the outline button with icon it is still important to set the height/width i guess to make it squre ( as most icons are not square). I'll look into a new button variant